Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.html

```diff
@@ -67,7 +67,7 @@
         <mat-list-item>
           <button
             mat-menu-item
-            (click)="sidenav.close(); releaseInfoService.displayReleaseDialog()"
+            (click)="sidenav.close(); dialogService.displayDialog('release-notes')"
```

Comment:
please use the same function as before or remove the `releaseInfoService.displayReleaseDialog()` function.